ampd/src/block_height_monitor.rs

```diff
@@ -47,8 +54,8 @@ impl<T: TmClient + Sync> BlockHeightMonitor<T> {
         loop {
             select! {
                 _ = interval.tick() => {
-                    let latest_block = self.client.latest_block().await.change_context(BlockHeightMonitorError::LatestBlockError)?;
-                    self.latest_height.store(latest_block.block.header.height.into(), std::sync::atomic::Ordering::SeqCst);
+                    let latest_block = self.client.latest_block().await.change_context(BlockHeightMonitorError::LatestBlock)?;
+                    self.latest_height_tx.send(latest_block.block.header.height.into()).change_context(BlockHeightMonitorError::Publish)?;
```

Comment:
Does it make more sense to panic here? From the doc, it seems imposibble to fail here since `self` alwast holds an instance of the receiver.

Reply:
yeah I think it makes more sense to panic, I changed it